src/blueprints/tv2/helpers/tv2-blueprint-configuration-mapper.ts

```diff
@@ -111,7 +111,7 @@ export class Tv2BlueprintConfigurationMapper {
     }
   }
 
-  private mapDveConfigurations(coreDveStyles: CoreDveStyle[]): DveConfiguration[] {
+  private mapSplitScreenConfigurations(coreDveStyles: CoreDveStyle[]): SplitScreenConfiguration[] {
```

Comment:
In reference to the comment above, do we then keep `coreDveStyles` because of this, or should both be renamed?

Reply:
I have renamed the `coreDveX` variables and interfaces to `coreSplitScreenX`.